Lucidtech/Las/Client.cs

```diff
@@ -531,6 +541,116 @@ public object CreateBatch(string? name = null, string? description = null)
             return ExecuteRequestResilient(RestSharpClient, request);
         }
 
+
+        /// <summary>
+        /// Create a dataset handle, calls the POST /datasets endpoint.
+        /// </summary>
+        /// <example>
+        /// Create a new dataset with the provided description.
+        /// on the document specified by '&lt;datasetId&gt;'
+        /// <code>
+        /// Client client = new Client();
+        /// var response = client.CreateDataset("Data gathered from the Mars Rover Invoice Scan Mission");
+        /// </code>
+        /// </example>
+        /// <param name="name">Name of the dataset</param>
+        /// <param name="description"> A brief description of the purpose of the dataset </param>
+        /// <returns>
+        /// A deserialized object that can be interpreted as a Dictionary with the fields datasetId and description.
+        /// datasetId can be used as an input when posting documents to make them a part of this dataset.
+        /// </returns>
+        public object CreateDataset(string? name = null, string? description = null)
```

Comment:
Isn't name and description usually provided in an 'attributes' object?

Reply:
Current code is a bit inconsistent, but CreateBatch had name and description provided explicitly, so I would suggest we do the same here for now.